modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/entry/util/test/ObjectEntryValuesUtilTest.java

```diff
@@ -0,0 +1,165 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.entry.util.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.object.constants.ObjectFieldConstants;
+import com.liferay.object.entry.util.ObjectEntryValuesUtil;
+import com.liferay.object.field.builder.TextObjectFieldBuilder;
+import com.liferay.object.model.ObjectField;
+import com.liferay.portal.kernel.test.rule.AggregateTestRule;
+import com.liferay.portal.kernel.test.util.RandomTestUtil;
+import com.liferay.portal.kernel.util.HashMapBuilder;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.vulcan.util.LocalizedMapUtil;
+
+import java.util.Collections;
+import java.util.Map;
+
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Yuri Monteiro
+ */
+@RunWith(Arquillian.class)
+public class ObjectEntryValuesUtilTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new LiferayIntegrationTestRule();
+
+	@Test
+	public void testGetTitleFieldValue() {
+		ObjectField objectField = _createObjectField("titleField");
+
+		Map<String, Object> modelAttributes1 = Collections.singletonMap(
+			"titleField", RandomTestUtil.randomString());
+
+		Object value1 = ObjectEntryValuesUtil.getTitleFieldValue(
+			ObjectFieldConstants.BUSINESS_TYPE_TEXT, modelAttributes1,
+			objectField, null, null);
+
+		Assert.assertNull(
+			"Should return null when values map is null, even if " +
+				"modelAttributes has data",
```

Comment:
Where did you see this pattern of adding messages to Assert in Liferay?

I've seen comments used to differentiate scenarios, but I've never seen custom messages added to the assertions.

Can you remove them?